src/components/sdk/MeetingControls/VideoInputBackgroundBlurControl.tsx

```diff
@@ -0,0 +1,171 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import {
+  DefaultVideoTransformDevice,
+  Device,
+  isVideoTransformDevice,
+  VideoTransformDevice,
+} from 'amazon-chime-sdk-js';
+import React, { ReactNode, useEffect, useState } from 'react';
+
+import { useBackgroundBlur } from '../../../providers/BackgroundBlurProvider';
+import { useVideoInputs } from '../../../providers/DevicesProvider';
+import { useLocalVideo } from '../../../providers/LocalVideoProvider';
+import { useMeetingManager } from '../../../providers/MeetingProvider';
+import { DeviceConfig } from '../../../types';
+import {
+  isOptionActive,
+  videoInputSelectionToDevice,
+} from '../../../utils/device-utils';
+import { ControlBarButton } from '../../ui/ControlBar/ControlBarButton';
+import { Camera, Spinner } from '../../ui/icons';
+import PopOverItem from '../../ui/PopOver/PopOverItem';
+import PopOverSeparator from '../../ui/PopOver/PopOverSeparator';
+
+interface Props {
+  /** The label that will be shown for video input control, it defaults to `Video`. */
+  label?: string;
+  /** The label that will be shown when the background blur checkbox is checked */
```

Comment:
nit: Can you please check and end all the doc statements with a "dot" for consistency? Can be done as a follow-up though.